packages/react/src/patterns/blocks/FeaturedLink/FeaturedLink.js

```diff
@@ -27,7 +25,7 @@ const { prefix } = settings;
 const FeaturedLink = ({ heading, card }) => {
   const cardProps = {
     ...card,
-    cta: { ...card.cta, icon: { src: ArrowRight20, ...card.cta.icon } },
+    cta: { ...card.cta, icon: { src: card.cta.src, ...card.cta.icon } },
```

Comment:
I think this should be `card.cta.icon.src`.

Reply:
argh yep